docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1765,291 +1765,347 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0063W":"Stack trace follows\n%s
+### ZWED0063W
+
+  Stack Trace Exception
 
   **Reason:**
 
-  TODO
+  This exception originates from the web socket and the stack trace message handles the generated exception. 
+  The stack trace for an exception helps in understanding the error and what went wrong with the code.
 
   **Action:**
 
-  TODO
+  No action needed unless user is experiencing an interruption in server service, then send stack to developers
 
 
 
-### ZWED0064W":"%s: invalid method %s
+### ZWED0064W
+
+  Invalid method
 
   **Reason:**
 
-  TODO
+  It will throw the warning if method is invalid (different from these methods: get|post|put|delete|ws)
 
   **Action:**
 
-  TODO
+  Review the warning message and use correct method
+
 
 
+### ZWED0065W
 
-### ZWED0065W":"Library %s is missing libraryVersion attribute for hosting files. Skipping file hosting.
+  Library plugin identifier is missing libraryVersion attribute for hosting files. Skipping file hosting.
 
   **Reason:**
 
-  TODO
+  Library plugin's definition is missing the library version attribute
 
   **Action:**
 
-  TODO
+  Add the correct library version in the plugin definition
 
 
 
-### ZWED0066W":"%s: getCapabilities() is not a function
+### ZWED0066W
+
+  pluginID: getCapabilities() is not a function
 
   **Reason:**
 
-  TODO
+  Handler _pluginid_ has no function/method defined as _getCapabilities()_
 
   **Action:**
 
-  TODO
+  Function _getCapabilities()_ should be defined
 
 
 
-### ZWED0068W":"Failed to set proxy authorizations. Error=
+### ZWED0068W
+
+  Failed to set proxy authorizations. Error=_ErrorMsg_
 
   **Reason:**
 
-  TODO
+  When handler fails to set proxy authorizations with handler pluginId, then it throws _ErrorMsg_ in the catch block.
```

Comment:
pluginId is not a variable used in the log message. Maybe it would be a better error message with that context, I agree, but it's not there so don't add it in. Reason should be like
"Failed to add proxy authorization with reason _errorMessage_"